tests/TestCase/Database/Expression/IntervalExpressionTest.php

```diff
@@ -0,0 +1,159 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ *
+ * Licensed under The Open Group Test Suite License
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ * @link          https://cakephp.org CakePHP(tm) Project
+ * @since         4.1.0
+ * @license       https://opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Cake\Test\TestCase\Database\Expression;
+
+use Cake\Database\Expression\DateTimeIntervalExpression;
+use Cake\Database\Expression\IntervalExpression;
+use Cake\Database\ExpressionInterface;
+use Cake\Database\Query;
+use Cake\Database\Type;
+use Cake\Database\ValueBinder;
+use Cake\Datasource\ConnectionManager;
+use Cake\I18n\FrozenTime;
+use Cake\TestSuite\TestCase;
+use DateInterval as CakeDateInterval;
```

Comment:
Why is this alias being used?

Reply:
Corrected. Was left over from a previous test though not sure how it got on the base class. It does remind me that I was going to ask and make sure (question posted on the file review you did below).